editor/plugins/script_text_editor.cpp

```diff
@@ -662,6 +662,10 @@ void ScriptTextEditor::_update_errors() {
 		errors_panel->pop(); // Indent.
 	}
 
+	if (line_number_gutter == -1) {
```

Comment:
It might be worth adding a comment to describe why this early return is here.

Reply:
The fix changed completely. The current fix is a bit more self explanatory.